app/views/layouts/_navbar.html.erb

```diff
@@ -59,6 +59,11 @@
 						<%= link_to "Account", edit_user_path(current_user), :title => "View your account info" %>
 					</li>
 					<% end %>
+					<% if @course and @cud %>
+					<li role="presentation">
+						<%= link_to "Course Profile", course_course_user_datum_path(@course, @cud), :title => "View your user profile for the current course" %>
```

Comment:
The only reason why I figured out how to add this was because I saw it referenced somewhere else in the project, but I couldn't find where this method (`course_course_user_datum_path`) was defined. My hunch is that Rails is auto-generating this method, but I would love clarification!